### DISCUSSION TITLE
Inconsistent tracking on Pi 3B

### DISCUSSION THREAD

**Discussion by LeoLTM** [mKeRix/room-assistant#373] [Q&A]
The BLE tracking is not working very reliable/consistent and because of that my lights are turning off and on the whole time, although my phone sits right next to me (2m away from the Pi) the whole time...

**Comment by zigt1301** [comment]
There have been similar posts about this issue. Have you tried reading those? [Watch changes rooms frequently while i stay still ](https://github.com/mKeRix/room-assistant/discussions/346) and [Entities configuration integrations Config Keys #342 ](https://github.com/mKeRix/room-assistant/discussions/342)
